Installer-Linux/linux-hosts-installer.sh

```diff
@@ -5,6 +5,15 @@
 # Copyright - Mitchell Krog - mitchellkrog@gmail.com 
 # https://github.com/mitchellkrogza
 
+#
+# root has to run the script
+#
+if [ `whoami` != "root" ]
```

Comment:
`if [[ $(id -u -n) != "root" ]]` might be better.

Reply:
isn't whoami a global Linux command nowadays? I have used this call for a long time and if I recalls it right (wayback in the brain if even) isn't there something about that using whoami is a better choice on sudo'ed systems?

Reply:
I agree with you @spirillen but, we are on a different scale. What we call Linux here, might be different from a computer to another, from distribution to another and so on ... And who knows maybe a really old box in this world is fetching data from here.